integration-cli/docker_api_images_test.go

```diff
@@ -23,59 +24,53 @@ func (s *DockerSuite) TestApiImagesFilter(c *check.C) {
 		v := url.Values{}
 		v.Set("filter", filter)
 		status, b, err := sockRequest("GET", "/images/json?"+v.Encode(), nil)
-		c.Assert(err, check.IsNil)
-		c.Assert(status, check.Equals, http.StatusOK)
+		c.Assert(err, checker.IsNil)
+		c.Assert(status, checker.Equals, http.StatusOK)
 
 		var images []image
-		if err := json.Unmarshal(b, &images); err != nil {
-			c.Fatal(err)
-		}
+		err = json.Unmarshal(b, &images)
+		c.Assert(err, checker.IsNil)
 
 		return images
 	}
 
 	errMsg := "incorrect number of matches returned"
-	if images := getImages("utest*/*"); len(images[0].RepoTags) != 2 {
-		c.Fatal(errMsg)
-	}
-	if images := getImages("utest"); len(images[0].RepoTags) != 1 {
-		c.Fatal(errMsg)
-	}
-	if images := getImages("utest*"); len(images[0].RepoTags) != 1 {
-		c.Fatal(errMsg)
-	}
-	if images := getImages("*5000*/*"); len(images[0].RepoTags) != 1 {
-		c.Fatal(errMsg)
-	}
+	images := getImages("utest*/*")
+	c.Assert(images[0].RepoTags, checker.HasLen, 2, check.Commentf(errMsg))
+
+	images = getImages("utest")
+	c.Assert(images[0].RepoTags, checker.HasLen, 1, check.Commentf(errMsg))
+
+	images = getImages("utest*")
+	c.Assert(images[0].RepoTags, checker.HasLen, 1, check.Commentf(errMsg))
+
+	images = getImages("*5000*/*")
+	c.Assert(images[0].RepoTags, checker.HasLen, 1, check.Commentf(errMsg))
```

Comment:
I think this whole block is better served by removing the newlines and putting a regular comment on the block rather than dealing with the complexity of a `Commentf`.